build/build_test.go

```diff
@@ -6,12 +6,23 @@ import (
 	"testing"
 )
 
+/*
+// TestBuild function
 func TestBuild(t *testing.T) {
 	got := build.Runfile()
 	want := "You successfully ran a Linux command from Go!!!" + "\n"
```

Comment:
We can add the newline as part of the string, without having to concatenate them:

```suggestion
	want := "You successfully ran a Linux command from Go!!!\n"
```

When we want to write longer multi-line strings in Go, we can use the backtick syntax to create a 'raw string':

```go
want := `this is a string which
continues over multiple lines
and the embedded newlines will
be preserved.\n`
```

Reply:
Noted and updated!